src/Security/MemberAuthenticator/SessionAuthenticationHandler.php

```diff
@@ -59,7 +60,7 @@ public function authenticateRequest(HTTPRequest $request)
             return null;
         }
         /** @var Member $member */
-        $member = Member::get()->byID($id);
+        $member = DataObject::get_by_id("SilverStripe\\Security\\Member", $id);
```

Comment:
```suggestion
        $member = Member::get_by_id($id);
```

Reply:
@michalkleiner  i don't want to be nitpicking here, but do we agree that Member::get_by_id is never called in the whole codebase (at least, in ss 4.13) ? if the team wants to replace that at some point, maybe better to make a whole sweep ;-)

Reply:
All good, either works. My point, from the isolated PR perspective, was to change the method in use, not on which class it's called, but functionally it's the same.

The most used is in fact
`$member = DataObject::get_by_id(Member::class, $id);`
and since we have the class already in `use`, we can do that instead of the string that is only used once in Permissions.

For sure we can do a global sweep, we can also do it gradually.

Reply:
But despite the above, I'd like to say thanks for raising these small improvement PRs, overall I think they will make a good measurable impact! So I'll try to be more trusting and focus on the bigger picture here ;-)

Reply:
@michalkleiner ;-) i've just used the same as in Permission::groupList but indeed i think there should be one preferred syntax for these and basically deprecate the rest

the changes i've made in my recent PR are actually doing quite a lot of difference in terms of page load :-) i hope the team can review them soon

Reply:
I agree with the original suggestion in this thread. Changing other usage of `get_by_id` is obviously not in scope but it's best practice to call methods on the appropriate class. 

At the _very very_ least use `::class` syntax instead of a hard coded string.

Reply:
@GuySartorelli agreed, let's go with Member::class. i'll make the change for Permission:groupsList as well so that at least we end up with something consistent ;-)